lib/control-connection.js

```diff
@@ -141,7 +245,7 @@ ControlConnection.prototype.getConnection = function (callback) {
  * it's the first time that trying to retrieve host information
  * @param {Function} callback
  */
-ControlConnection.prototype.initOnConnection = function (firstTime, callback) {
+ControlConnection.prototype.refreshOnConnection = function (firstTime, callback) {
   var c = this.connection;
   var self = this;
   self.log('info', 'Connection acquired, refreshing nodes list');
```

Comment:
I couldn't seem to reproduce this, so perhaps this is not a problem (though I'd like to understand why)